docs/build-instructions/windows.md

```diff
@@ -73,19 +69,19 @@ If none of this works, do install Github Desktop and use its Git shell. Makes li
     * https://github.com/TooTallNate/node-gyp/issues/297
     * https://code.google.com/p/gyp/issues/detail?id=393
 
-* `script/build` stops at installing runas with 'Failed at the runas@x.y.z install script.'
+* `script/build` stops at installing runas with `Failed at the runas@x.y.z install script.`
 
-  See the next item.
+  * See the next item.
 
 * `error MSB8020: The build tools for Visual Studio 2010 (Platform Toolset = 'v100') cannot be found.`
 
-  * If you're building atom with Visual Studio 2013 try executing the following
-    command in your Git shell and then re-run `script/build`:
+  * If you're building Atom with Visual Studio 2013 or above make sure the `GYP_MSVS_VERSION` environment variable is set, and then re-run `script/build`:
 
+    ```bash
+    $env:GYP_MSVS_VERSION=2013 # 2015 if using Visual Studio 2015, and so on
```

Comment:
> EDIT: For what it's worth I don't think Github Desktop set up my Git Shell correctly so I've been using the Git Bash (MINGW64???) provided by Git for Windows, so that could be the problem

Yup, that is the problem